istio/assets/dashboards/istio_1_5_overview

```diff
@@ -315,7 +341,7 @@
             "tile_def": {
                 "requests": [
                     {
-                        "q": "top(avg:istio.grpc.server.msg_sent_total{*} by {host}.as_count(), 10, 'mean', 'desc'), top(avg:istio.grpc.server.msg_received_total{*} by {host}.as_count(), 10, 'mean', 'desc')",
+                        "q": "top(avg:istio.grpc.server.msg_sent_total{*}.as_count(), 10, 'mean', 'desc'), top(avg:istio.grpc.server.msg_received_total{*}.as_count(), 10, 'mean', 'desc')",
```

Comment:
Any reason we don't group by `host` anymore?

Same for other cases.

Reply:
The metrics from the istiod endpoints don't have a `host` tag when they're collected by cluster agent.